data/hai/hai reveal 4 middle.txt

```diff
@@ -1114,46 +1140,190 @@ mission "Hai Reveal [A09] History of Appeasement"
+						not "Hai Reveal [A09] History of Appeasement: label types"
+				`	"Does Hai media not deal with this sort of thing?"`
+					goto media
+					to display
+						has "Hai Reveal [A09] History of Appeasement: label fix"
+						not "Hai Reveal [A09] History of Appeasement: label media"
+				# End discussion.
+				`	"We've discussed enough. Let's get to work."`
+					goto go
+					to display
+						has "Hai Reveal [A09] History of Appeasement: label fix"
+
+			# First phase of conversation: listen in or interrupt.
+			label listen
+			action
+				set "Hai Reveal [A09] History of Appeasement: label listen"
+			`	It is difficult to hear precisely without making yourself known, so you struggle to understand. At first, Teeneep is annoyed, maybe saying words like: pirate, victim, orchestra, and betrayal. Sayari's responses revolve around: necessity, an angry child, guilt, and eggs. Something sets off Teeneep, and she keeps screaming "tweezers." Sayari puts a paw gently on Teeneep's shoulders and says in a soothing maternal tone, "green the moon blue." At this point, it becomes clear that the conversation is entirely in Hai, which you don't speak, and you have no idea what the two women are actually discussing.`
+				goto questions
+			label childish
+			action
+				set "Hai Reveal [A09] History of Appeasement: interrupted"
+			`	As the pair turn towards you, Teeneep looks mortified. "Captain <last>, I... I'm sorry you had to see me like this. I just... Sayari?"`
+			`	"Welcome back, Captain <last>. I must apologize for my friend's state. She's just discovered some rather unfortunate information."`
+				goto arrived
+			label interrupt
+			action
+				set "Hai Reveal [A09] History of Appeasement: interrupted"
+			`	Realizing that you've arrived, Sayari tilts her head slightly to one side to indicate she's speaking to you, while not taking her eyes off Teeneep. "Welcome back, Captain <last>. I must apologize for my friend's state. She's just discovered some rather unfortunate information."`
+				goto arrived
+			label arrived
+			`	"Unfortunate!" Teeneep very nearly shrieks the word. "They've been doing this for decades! Decades, Sayari! Tracking down and ultimately paying off pirates with our technology to leave us alone, even when they've hurt, killed, kidnapped, and enslaved our people!`
+			`	"You knew! This whole time, for a half century, you knew, and you did nothing!"`
+			`	Teeneep closes her eyes and breathes deeply until she has calmed herself down. It looks like this is a technique she's practiced many times.`
+			`	The three of you sit down once Teeneep looks ready to talk.`
+				goto questions
+			label friends
+			action
+				set "Hai Reveal [A09] History of Appeasement: label friends"
+			`	Sayari looks at you apologetically. "She's always been quite a spark, but we usually get along very well."`
+			`	Teeneep adds, "Sayari has been my close friend and mentor since I was a head shorter than her."`
```

Comment:
This order feels more appropriate to me.
```suggestion
			`	Teeneep adds, "Sayari has been my mentor and ultimately a close friend since I was a head shorter than her."`
```

Reply:
You're mixing past tense and present tense in a way that is quite confusing. I changed it to, "Sayari has been my mentor since I was a head shorter than her. I consider her my closest and most trusted friend."